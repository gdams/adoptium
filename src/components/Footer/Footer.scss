@media only screen and (width <= 480px) {
  footer {

    text-align: center;

    h5 {
      padding: .5em;
    }
  }
}
