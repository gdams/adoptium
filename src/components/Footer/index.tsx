import React from 'react';
import { FaYoutube, FaGithub, FaSlack, FaLinkedin } from 'react-icons/fa';
import { FaXTwitter } from 'react-icons/fa6';
import RandomContributor from '../RandomContributor';
import LanguageSelector from '../LanguageSelector';
import { Link, Trans, useTranslation } from 'gatsby-plugin-react-i18next';
import LeavingSiteDisclaimerModal from '../LeavingSiteDisclaimerModal';
import './Footer.scss';

const Footer = (): JSX.Element => {

  const {t} = useTranslation();

  return (
    <>
    <section className="bottom-info">
      <RandomContributor />
      <LeavingSiteDisclaimerModal />
    </section>
      <div className="bg-grey">
        <div className="container mt-5">
          <footer className="row row-cols-4 py-5 border-top">

            <div className="col-sm-3 col-12">
              <p className="h5 pb-1"><Trans i18nKey='footer.eclipse.foundation' defaults='Eclipse Foundation'/></p>
              <ul className="nav flex-column">
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.about.us' defaults='About Us'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/foundation/contact.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.contact.us' defaults='Contact Us'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/donate/adoptium" className="nav-link p-0 text-muted"><Trans i18nKey='footer.donate' defaults='Donate'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/membership" className="nav-link p-0 text-muted"><Trans i18nKey='footer.members' defaults='Members'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/documents/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.governance' defaults='Governance'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.code.of.conduct' defaults='Code of Conduct'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/artwork/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.logo.and.artwork' defaults='Logo and Artwork'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/foundation/directors.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.board.of.directors' defaults='Board of Directors'/></a></li>
              </ul>
            </div>

            <div className="col-sm-3 col-12">
            <p className="h5 pb-1"><Trans i18nKey='footer.legal' defaults='Legal'/></p>
              <ul className="nav flex-column">
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/legal/privacy.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.privacy.policy' defaults='Privacy Policy'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/legal/termsofuse.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.terms.of.use' defaults='Terms of Use'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/legal/copyright.php" className="nav-link p-0 text-muted"><Trans i18nKey='footer.copyright.agent' defaults='Copyright Agent'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/legal/epl-2.0/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.eclipse.public.license' defaults='Eclipse Public License'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/legal/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.legal.resources' defaults='Legal Resources'/></a></li>
              </ul>
            </div>

            <div className="col-sm-3 col-12">
            <p className="h5 pb-1"><Trans i18nKey='footer.useful.links' defaults='Useful Links'/></p>
              <ul className="nav flex-column">
                <li className="nav-item mb-2"><a href="https://github.com/adoptium/adoptium-support/issues" className="nav-link p-0 text-muted"><Trans i18nKey='footer.report.a.bug' defaults='Report a Bug'/></a></li>
                <li className="nav-item mb-2"><Link to="/docs" className="nav-link p-0 text-muted"><Trans i18nKey='footer.documentation' defaults='Documentation'/></Link></li>
                <li className="nav-item mb-2"><Link to="/contributing" className="nav-link p-0 text-muted"><Trans i18nKey='footer.how.to.contribute' defaults='How to Contribute'/></Link></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/mail/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.mailing.lists' defaults='Mailing Lists'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/forums/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.forums' defaults='Forums'/></a></li>
                <li className="nav-item mb-2"><a href="https://marketplace.eclipse.org/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.marketplace' defaults='Marketplace'/></a></li>
                <li className="nav-item mb-2"><a href="https://eclipse-foundation.store/collections/eclipse-adoptium" className="nav-link p-0 text-muted"
                  data-bs-toggle="modal"
                  data-bs-target="#leavingSiteDisclaimerModal"
                  data-bs-message={t('swag.store.disclaimer', 'By clicking the continue button, you will leave our website. Please be aware that new terms of use will apply to the Eclipse Foundation store, powered by Fourthwall: https://eclipse-foundation.store/.')}
                  data-bs-location="https://eclipse-foundation.store/collections/eclipse-adoptium"
                  ><Trans i18nKey='footer.swag.store' defaults='Swag Store'/></a></li>
              </ul>
            </div>

            <div className="col-sm-3 col-12">
            <p className="h5 pb-1"><Trans i18nKey='footer.useful.other' defaults='Other'/></p>
              <ul className="nav flex-column">
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/ide/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.ide.and.tools' defaults='IDE and Tools'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/projects" className="nav-link p-0 text-muted"><Trans i18nKey='footer.projects' defaults='Projects'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/workinggroups/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.working.groups' defaults='Working Groups'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/org/research/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.research.eclipse' defaults='Research@Eclipse'/></a></li>
                <li className="nav-item mb-2"><a href="https://www.eclipse.org/security/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.report.a.vulnerability' defaults='Report a Vulnerability'/></a></li>
                <li className="nav-item mb-2"><a href="https://status.eclipse.org/" className="nav-link p-0 text-muted"><Trans i18nKey='footer.service.status' defaults='Service Status'/></a></li>
              </ul>
            </div>

            <div className="col-sm-6 col-12 py-2 d-flex">
              <span className="text-muted"><Trans i18nKey='footer.all.rights.reserved' defaults='Copyright Â© Eclipse Foundation. All Rights Reserved.'/></span>
            </div>
            <ul className="nav col-md-5 col-9 pb-4 justify-content-end list-unstyled d-flex">
              <li className="ms-3"><a href="https://www.netlify.com"> <img src="/images/netlify-light.svg" alt="Deploys by Netlify" style={{height: '25px'}}/> </a></li>
              <li className="ms-3"><a className="text-muted" aria-label="Adoptium Twitter Account" href="https://x.com/adoptium"><FaXTwitter size={25} /></a></li>
              <li className="ms-3"><a className="text-muted" aria-label="Adoptium Linkedin Account" href="https://www.linkedin.com/showcase/adoptium/"><FaLinkedin size={25} /></a></li>
              <li className="ms-3"><a className="text-muted" aria-label="Adoptium YouTube Account" href="https://www.youtube.com/c/EclipseAdoptium"><FaYoutube size={25} /></a></li>
              <li className="ms-3"><a className="text-muted" aria-label="Adoptium GitHub Account" href="https://github.com/adoptium"><FaGithub size={25} /></a></li>
              <li className="ms-3"><Link className="text-muted" aria-label="Adoptium Slack Account" to="/slack"><FaSlack size={25} /></Link></li>
            </ul>
            <div className="col-12 d-flex ">
              <span className="text-muted"><Trans i18nKey='footer.trademarks' defaults='Java and OpenJDK are trademarks or registered trademarks of Oracle and/or its affiliates. Other names may be trademarks of their respective owners.'/></span>
            </div>
          </footer>
        </div>
      </div>
      <LanguageSelector />
    </>
  );
};

export default Footer;
