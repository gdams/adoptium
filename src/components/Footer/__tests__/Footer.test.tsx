import React from 'react';
import { render } from '@testing-library/react';
import Footer from '..';
import '../../../../test/__mocks__/intersectionObserverMock';

describe('Footer component', () => {
  it('renders correctly', () => {
    const { container } = render(
      <Footer />
    );
    expect(container).toMatchSnapshot();
  });
});