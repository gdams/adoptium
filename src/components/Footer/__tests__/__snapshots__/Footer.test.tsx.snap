// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > renders correctly 1`] = `
<div>
  <section
    class="bottom-info"
  >
    <div
      class="random-contributor"
    />
  </section>
  <div
    class="bg-grey"
  >
    <div
      class="container mt-5"
    >
      <footer
        class="row row-cols-4 py-5 border-top"
      >
        <div
          class="col-sm-3 col-12"
        >
          <p
            class="h5 pb-1"
          >
            Text
          </p>
          <ul
            class="nav flex-column"
          >
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/foundation/contact.php"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/donate/adoptium"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/membership"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/documents/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/artwork/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/foundation/directors.php"
              >
                Text
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-sm-3 col-12"
        >
          <p
            class="h5 pb-1"
          >
            Text
          </p>
          <ul
            class="nav flex-column"
          >
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/legal/privacy.php"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/legal/termsofuse.php"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/legal/copyright.php"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/legal/epl-2.0/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/legal/"
              >
                Text
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-sm-3 col-12"
        >
          <p
            class="h5 pb-1"
          >
            Text
          </p>
          <ul
            class="nav flex-column"
          >
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://github.com/adoptium/adoptium-support/issues"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="/docs"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="/contributing"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/mail/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/forums/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://marketplace.eclipse.org/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://store.adoptium.net/"
              >
                Text
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-sm-3 col-12"
        >
          <p
            class="h5 pb-1"
          >
            Text
          </p>
          <ul
            class="nav flex-column"
          >
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/ide/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/projects"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/workinggroups/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/org/research/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://www.eclipse.org/security/"
              >
                Text
              </a>
            </li>
            <li
              class="nav-item mb-2"
            >
              <a
                class="nav-link p-0 text-muted"
                href="https://status.eclipse.org/"
              >
                Text
              </a>
            </li>
          </ul>
        </div>
        <div
          class="col-sm-6 col-12 py-2 d-flex"
        >
          <span
            class="text-muted"
          >
            Text
          </span>
        </div>
        <ul
          class="nav col-md-5 col-9 pb-4 justify-content-end list-unstyled d-flex"
        >
          <li
            class="ms-3"
          >
            <a
              href="https://www.netlify.com"
            >
               
              <img
                alt="Deploys by Netlify"
                src="/images/netlify-light.svg"
                style="height: 25px;"
              />
               
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              aria-label="Adoptium Twitter Account"
              class="text-muted"
              href="https://x.com/adoptium"
            >
              <svg
                fill="currentColor"
                height="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 512 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M389.2 48h70.6L305.6 224.2 487 464H345L233.7 318.6 106.5 464H35.8L200.7 275.5 26.8 48H172.4L272.9 180.9 389.2 48zM364.4 421.8h39.1L151.1 88h-42L364.4 421.8z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              aria-label="Adoptium Linkedin Account"
              class="text-muted"
              href="https://www.linkedin.com/showcase/adoptium/"
            >
              <svg
                fill="currentColor"
                height="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              aria-label="Adoptium YouTube Account"
              class="text-muted"
              href="https://www.youtube.com/c/EclipseAdoptium"
            >
              <svg
                fill="currentColor"
                height="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 576 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M549.655 124.083c-6.281-23.65-24.787-42.276-48.284-48.597C458.781 64 288 64 288 64S117.22 64 74.629 75.486c-23.497 6.322-42.003 24.947-48.284 48.597-11.412 42.867-11.412 132.305-11.412 132.305s0 89.438 11.412 132.305c6.281 23.65 24.787 41.5 48.284 47.821C117.22 448 288 448 288 448s170.78 0 213.371-11.486c23.497-6.321 42.003-24.171 48.284-47.821 11.412-42.867 11.412-132.305 11.412-132.305s0-89.438-11.412-132.305zm-317.51 213.508V175.185l142.739 81.205-142.739 81.201z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              aria-label="Adoptium GitHub Account"
              class="text-muted"
              href="https://github.com/adoptium"
            >
              <svg
                fill="currentColor"
                height="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 496 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                />
              </svg>
            </a>
          </li>
          <li
            class="ms-3"
          >
            <a
              aria-label="Adoptium Slack Account"
              class="text-muted"
              href="/slack"
            >
              <svg
                fill="currentColor"
                height="25"
                stroke="currentColor"
                stroke-width="0"
                viewBox="0 0 448 512"
                width="25"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M94.12 315.1c0 25.9-21.16 47.06-47.06 47.06S0 341 0 315.1c0-25.9 21.16-47.06 47.06-47.06h47.06v47.06zm23.72 0c0-25.9 21.16-47.06 47.06-47.06s47.06 21.16 47.06 47.06v117.84c0 25.9-21.16 47.06-47.06 47.06s-47.06-21.16-47.06-47.06V315.1zm47.06-188.98c-25.9 0-47.06-21.16-47.06-47.06S139 32 164.9 32s47.06 21.16 47.06 47.06v47.06H164.9zm0 23.72c25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06H47.06C21.16 243.96 0 222.8 0 196.9s21.16-47.06 47.06-47.06H164.9zm188.98 47.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06s-21.16 47.06-47.06 47.06h-47.06V196.9zm-23.72 0c0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06V79.06c0-25.9 21.16-47.06 47.06-47.06 25.9 0 47.06 21.16 47.06 47.06V196.9zM283.1 385.88c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06-25.9 0-47.06-21.16-47.06-47.06v-47.06h47.06zm0-23.72c-25.9 0-47.06-21.16-47.06-47.06 0-25.9 21.16-47.06 47.06-47.06h117.84c25.9 0 47.06 21.16 47.06 47.06 0 25.9-21.16 47.06-47.06 47.06H283.1z"
                />
              </svg>
            </a>
          </li>
        </ul>
        <div
          class="col-12 d-flex "
        >
          <span
            class="text-muted"
          >
            Text
          </span>
        </div>
      </footer>
    </div>
  </div>
  <div
    class="App lngg"
  >
    <form
      class=""
    >
      <div
        class="dropdown"
      >
        <button
          aria-expanded="false"
          aria-label="Language Selector"
          class="text-left text-white dropdown-toggle btn btn-primary"
          id="dropdown-flags"
          type="button"
        >
          Text
        </button>
      </div>
    </form>
  </div>
</div>
`;
